pkg/minikube/registry/drvs/podman/podman.go

```diff
@@ -107,6 +110,16 @@ func status() registry.State {
 			glog.Warningf("Warning ! minimum required version for podman is %s. your version is %q. minikube might not work. use at your own risk. To install latest version please see https://podman.io/getting-started/installation.html ", minReqPodmanVer.String(), v.String())
 		}
 
+		if v.GTE(podmanVerTwo) {
+			return registry.State{
+				Error:     fmt.Errorf("Podman2 is not supported yet"),
+				Installed: true,
+				Healthy:   false,
+				Fix:       "Install a compatible Podman driver",
```

Comment:
Would something more specific be better, similar to?:

```suggestion
				Fix:       fmt.Sprintf("Install a compatible Podman driver. %q+", minReqPodmanVer.String()),
```

Reply:
@Dean-Coakley I like the second way you suggested, it is better to tell the user what version to install

Reply:
I think the word "driver" should be replaced by "version":

Install a compatible Podman version.